assets/app/views/attachPVC.html

```diff
@@ -0,0 +1,172 @@
+<div class="content">
+  <div class="container">
+    <breadcrumbs breadcrumbs="breadcrumbs"></breadcrumbs>
+    <div class="row">
+      <div class="col-md-12">
+        <alerts alerts="alerts"></alerts>
+        <div ng-show="!pvcs || !(attach.deploymentConfig || attach.deployment)">Loading...</div>
+
+        <div ng-show="pvcs && !pvcs.length && (attach.deploymentConfig || attach.deployment)" class="empty-state-message empty-state-full-page">
+          <h2 class="text-center">No persistent volume claims.</h2>
+
+          <p class="gutter-top">
+            A <b>persistent volume claim</b> is required to attach to this 
+            <span ng-if="attach.deploymentConfig">deployment config,</span>
+            <span ng-if="attach.deployment">deployment,</span>
+            but none are loaded on this project.
+          </p>
+
+          <p>
+            To claim storage from a persistent volume, refer to the documentation on <a ng-href="{{'persistent_volumes' | helpLink}}">using persistent volumes</a>.
+          </p>
+
+          <p ng-if="attach.deploymentConfig"><a href="{{attach.deploymentConfig | navigateResourceURL}}">Back to deployment config</a></p>
+          <p ng-if="attach.deployment"><a href="{{attach.deployment | navigateResourceURL}}">Back to deployment</a></p>
```

Comment:
This is a nit, but if you got here from the browse page page, the link will take you back to a deployment config, not the pod.